### PR TITLE
clh: Direct IO support for block devices 

### PR DESCRIPTION
**Pull request by alex-matei**
Cloud hypervisor has support for direct i/o, this patch adds support for enabling it in kata. It doesn't know about noflush, removed it from the list of valid options for clh.
Firecracker supports noflush semantic via Unsafe cache type.
There is no support for direct i/o in firecracker, I removed it from config file.

Fixes: #7798 #7823

### CODE REVIEW

File: src/runtime/pkg/katautils/config.go [hunk 1]
```diff
@@ -1059,7 +1059,6 @@ func newClhHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		BlockDeviceDriver:              blockDriver,
 		BlockDeviceCacheSet:            h.BlockDeviceCacheSet,
 		BlockDeviceCacheDirect:         h.BlockDeviceCacheDirect,
-		BlockDeviceCacheNoflush:        h.BlockDeviceCacheNoflush,
```
Comment (amshinde) [inline src/runtime/pkg/katautils/config.go:1062]: Is this not supported for cloud-hypervisor? Also, the issue you mentioned on the `Fixes`  line #7823 talks about firecracker and not ccloud-hypervisor, so I was bit confused by that. In any case, will be useful to add this as a separate commit if the config needs to be removed explaining why it was removed.

**Review by jepio — APPROVED** [review]

**Comment by jepio** [comment]
@alex-matei please rebase the PR due to changes in CI.

**Comment by alex-matei** [comment]
@jepio Done, rebased.

**Review by amshinde — APPROVED** [review]

**Comment by jepio** [comment]
@alex-matei i'm sorry to have to ask this - but could you rebase again? all the changes to the ci setup is messing with required checks and so github is not letting me merge this.

**Comment by alex-matei** [comment]
Sure, no problem, rebased.
